rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_statement_wrapper.cpp

```diff
@@ -51,14 +51,14 @@ SqliteStatementWrapper::~SqliteStatementWrapper()
 std::shared_ptr<SqliteStatementWrapper> SqliteStatementWrapper::execute_and_reset()
 {
   int return_code = sqlite3_step(statement_);
-  if (!isQueryOk(return_code)) {
+  if (!is_query_ok(return_code)) {
```

Comment:
just as a remark. I know it's only single threaded for now, but I believe operations like that can benefit from atomic compare_exchange.